apps/zero-runtime-next-app/src/app/material-ui/react-avatar/page.tsx

```diff
@@ -0,0 +1,86 @@
+'use client';
```

Comment:
this page is generated from the script https://github.com/mui/material-ui/pull/41267/files#diff-a4d30ddb2faa2486c2c65415ce350349fb32db41f7467e0d5680ac41b41febd5